ui/apps/platform/src/Containers/NetworkGraph/common/FlowsTable.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
Fixes a bug where the list fails to update properly during filtering due to duplicate keys. Since some elements in the list are indistinguishable from each other, the `index` must be combined with the `id` to ensure uniqueness.